src/serializers/decoders/current/v15/decode_graph_entities.c

```
@@ -51,6 +54,30 @@ static SIValue _RdbLoadPoint
+	// .
+	// .
+	// key:value
+
+	uint n = SerializerIO_ReadUnsigned(rdb);
+	SIValue keys[n];
+	SIValue vals[n];
+
+	for(uint i = 0; i < n; i++) {
+		keys[i] = _RdbLoadSIValue(rdb);
+		vals[i] = _RdbLoadSIValue(rdb);
+	}
```

cppcheck

[error] Memory leak

(memleak)